### DISCUSSION TITLE
IP exit to Munich?

### DISCUSSION THREAD

**Discussion by ElRak1337** [Ironclad/rivet#175] [Q&A]
Hi there,
I have a question. Why does my firewall tell me that Rivet sends a singnal to Munich? What do you do with the data? Could you look at our temples?

**Comment by abrenneke** [comment]
I have no idea. Nothing of that sort has been configured. You're free to look at the source to confirm.

**Comment by ElRak1337** [comment]
![image](https://github.com/Ironclad/rivet/assets/130494728/096c1e52-0b31-423a-afb2-a0859f4cd892)

**Reply by abrenneke** [reply]
This looks like it's connecting to GitHub, so it's probably just the auto-update checker?
